cmd/docker-app/render.go

```diff
@@ -6,11 +6,15 @@ import (
 
 	"github.com/docker/app/internal"
 	"github.com/docker/app/internal/packager"
-	"github.com/docker/app/internal/renderer"
+	"github.com/docker/app/internal/render"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
+
+	_ "github.com/docker/app/internal/renderer/gotemplate"
```

Comment:
Why not calling these 3 lines directly in the render package? I think it is implementation detail and should hidden from the render library client point of view.

Reply:
@silvin-lubecki true, wasn't really sure where to put it :stuck_out_tongue: `internal/render` seems good too :angel: